backend/services/messageService.js

```diff
@@ -0,0 +1,21 @@
+const messageDAO = require('../DAO/messageDAO');
+
+module.exports = {
+
+    saveNewMessage: async (message) => {
+        return await messageDAO.saveNewMessage(message);
+    },
+
+    getConversation: async (studentId, emplyerId) => {
+        return await messageDAO.getConversation(studentId,emplyerId);
```

Comment:
Small typo in employerId, both in method signature and in the other method call

Reply:
ok I fixed this